eem-sparks/src/main/java/com/nice/eem/sparkathon/AWSS3ClientBuilder.java

```diff
@@ -1,53 +1,35 @@
 package com.nice.eem.sparkathon;
 
-import com.amazonaws.auth.AWSCredentials;
 import com.amazonaws.auth.AWSStaticCredentialsProvider;
 import com.amazonaws.auth.BasicAWSCredentials;
-import com.amazonaws.auth.InstanceProfileCredentialsProvider;
 import com.amazonaws.services.s3.AmazonS3;
 import com.amazonaws.services.s3.AmazonS3ClientBuilder;
 
-import lombok.extern.slf4j.Slf4j;
 
-import org.springframework.stereotype.Component;
-
-
-
-@Component
-@Slf4j
 public class AWSS3ClientBuilder {
 
 
+    public static void main(String[] args) {
+        buildS3Client();
+    }
 
 
     /**
      * build aws s3 client
      *
      * @return
      */
-    public AmazonS3 buildS3Client(S3Config s3Config) {
-        AWSCredentials awsCredentials;
+    public static AmazonS3 buildS3Client() {
         AmazonS3 s3Client;
-        String cloudDeployment = s3Config.getIsCloudDeployment();
-        String awsRegion = s3Config.getRegion();
-        if (cloudDeployment.equalsIgnoreCase("false")) {
-            log.info("Building aws s3 client for non-cloud deployment, isCloudDeployment = {}, aws region = {}", cloudDeployment, awsRegion);
-            String accessKey = s3Config.getAccessKey();
-            String secretKey = s3Config.getSecretKey();
-            awsCredentials = new BasicAWSCredentials(accessKey, secretKey);
-            s3Client = AmazonS3ClientBuilder
-                    .standard()
-                    .withCredentials(new AWSStaticCredentialsProvider(awsCredentials))
-                    .withRegion(awsRegion)
-                    .build();
-        } else {
-            log.info("Building aws s3 client for clould deployment, isCloudDeployment = {}, aws region = {}", cloudDeployment, awsRegion);
-            s3Client = AmazonS3ClientBuilder
-                    .standard()
-                    .withCredentials(new InstanceProfileCredentialsProvider(true))
-                    .withRegion(awsRegion)
-                    .build();
-        }
+        String awsRegion = "us-east-1";
+        BasicAWSCredentials awsCreds = new BasicAWSCredentials("XXXXXXXXXXXXXXXXXXXX", "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

If possible, use temporary security credentials (IAM roles) instead of long-term access keys.
Long-term access keys, such as those associated with IAM users and AWS account root users, remain valid until you manually revoke them. However, temporary security credentials obtained through IAM roles and other features of the AWS Security Token Service expire after a short period of time. Use temporary security credentials to help reduce your risk in case credentials are accidentally exposed.
Learn more about [best practices for managing AWS access keys](https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html).